test/components/entity/feed-entry.spec.js

```diff
@@ -163,10 +170,8 @@ describe('EntityFeedEntry', () => {
         entity: { uuid: 'e' }
       };
       if (options.submission === true) {
-        testData.extendedSubmissions.createPast(1);
-        details.submissionCreate = testData.extendedAudits
-          .createPast(1, { action: 'submission.create' })
-          .last();
+        const submission = testData.extendedSubmissions.createPast(1);
```

Comment:
```suggestion
        const submission = testData.extendedSubmissions.createPast(1).last();
```

`createPast()` returns the test data store for chaining. I guess in the case of `entity.create`, the component is just looking at the presence of the `submission` property and not its value, which is why this wasn't an issue for tests.